casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -236,43 +233,48 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
     "4! (4!)"   -> VOLATILE_EVENT             -> merges(F_, F1 | S1, S0),
     "4! (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(F_, C_ | S0, S0),
     "4X (4!)"   -> VOLATILE_EVENT             -> merges(F_, F_ | S0, Nil),
-    "4X (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(F_, C_ | S0, Nil),
-    "C! (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(C_, C_ | S0, S0),
-    "CX (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(C_, C_ | S0, Nil),
+    "4X (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> merges(F_, C_ | S0, Nil),
+    "C! (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(C_, C_ | S0, S0),
```

Comment:
interesting... This should merge, because `C! C!` merges

Reply:
oh I see why, the test case is wrong - should be `merges(C_, C1 | S1, S0)`, or the right C_ eats the base send as well

Reply:
Done